src/Components/test/E2ETest/Tests/WebAssemblyICUShardingTest.cs

```diff
@@ -39,7 +36,7 @@ public void LoadingApp_FrenchLanguage_Works()
             Assert.Equal(culture.ToString(), cultureDisplay.Text);
 
             var dateDisplay = Browser.Exists(By.Id("dateTime"));
-            Assert.Equal(DisplayTime.ToString(culture), dateDisplay.Text);
+            Assert.Equal("02/09/2020 00:00:00", dateDisplay.Text);
```

Comment:
The ICU data on the server depends on the OS. On my machine, it's up to date with WASM but not so on the build machines. Using hardcoded strings seems like the best way to proceed: https://github.com/dotnet/runtime/pull/42136